view/code/index.ts

```diff
@@ -88,6 +88,21 @@ function listenChanges(input: HTMLInputElement): void {
     }
   }
 
+  function processRgbBlur(): void {
+    let value = input.value.trim()
+    let parsed = parseAnything(value)
+
+    if (parsed && parsed.mode in formats.get()) {
```

Comment:
It will be better to save supported format once

Reply:
Moved all formats that can be obtained via `parsed.mode` into a separate object